pymilo/transporters/preprocessing_transporter.py

```diff
@@ -4,7 +4,13 @@
 from ..utils.util import check_str_in_iterable, get_sklearn_type
 from .transporter import AbstractTransporter, Command
 from .general_data_structure_transporter import GeneralDataStructureTransporter
+from .function_transporter import FunctionTransporter
+from scipy.interpolate._bsplines import BSpline
```

Comment:
`scipy` is getting import by scikit implicitly

Reply:
We need to add `scipy` to both `requirements.txt` and `dev-requirements.txt` because there is no guarantee that `scipy` will remain an essential requirement for `scikit-learn` in the future.

Reply:
fixed [4758dcf](https://github.com/openscilab/pymilo/pull/104/commits/4758dcfe873716c9f34417714750a5905a7aea73).